logic.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp2
+{
+    internal class logics
+    {
+        public void Seq1(List<int> arr, ref object start, ref object end, ref int i, ref int k)
```

Comment:
Naming of the methods can be improved, it needs to be intuitive to users so that the name itself gives to a large exent what the method is supposed to perform.